src/components/experiment-tracking/run-metadata/run-metadata.scss

```diff
@@ -12,6 +15,28 @@
   }
 }
 
+.details-metadata__title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.details-matadata__buttons {
```

Comment:
The alignment here seems ever so slightly off. 

![image](https://user-images.githubusercontent.com/16869061/145396519-8f18007b-13cb-4fa3-bb5a-806bf715f6a9.png)

I think they should be aligned to the baseline.

Reply:
I had made minor adjustments to tackle the tiny discrepancy - please see below. 
![image](https://user-images.githubusercontent.com/27775658/145903313-dccdcbed-46bc-40e4-acc5-2639f9d413b9.png). 

This seem to be caused by the setting of the svg which I've adjust this with a padding for now, please feel free to propose any better way to cater for this small visual change ( I just don't want to dwell on this as a blocker for the release).